hathor/transaction/static_metadata.py

```diff
@@ -32,6 +43,10 @@ class VertexStaticMetadata(ABC):
     This class is an abstract base class for all static metadata types that includes attributes common to all vertex
     types.
     """
+
+    # XXX: this is only used to defer the reward-lock verification from the transaction spending a reward to the first
```

Comment:
I didn't understand your comment. Can you elaborate, please?

Reply:
This comment is not mine, I only moved the existing description from `TransactionMetadata`:

https://github.com/HathorNetwork/hathor-core/blob/5e999598ea27550920a8b6a510249914dac96cbb/hathor/transaction/transaction_metadata.py#L48

Reply:
Ok. Thanks!